pkg/commands/git.go

```diff
@@ -488,12 +484,12 @@ func (c *GitCommand) GetCommits() []Commit {
 func (c *GitCommand) GetLog() string {
 	// currently limiting to 30 for performance reasons
 	// TODO: add lazyloading when you scroll down
-	result, err := c.OSCommand.RunCommandWithOutput("git log --oneline -30")
-	if err != nil {
-		// assume if there is an error there are no commits yet for this branch
-		return ""
+	if result, err := c.OSCommand.RunCommandWithOutput("git log --oneline -30"); err == nil {
+		return result
```

Comment:
The reason for originally returning "" was that otherwise we'd end up with this:
![image](https://user-images.githubusercontent.com/8456633/45663327-30c4a300-bb49-11e8-8d7a-eeae88f08eaf.png)

Reply:
yep it's still the behaviour, I reversed the logic to inline the error check

Reply:
I think we should opt to use `err != nil` over `err == nil` just for consistency, because I've just gotten confused from assuming that it was `err != nil` haha

Reply:
right, I wonder if I maybe did it elsewhere